clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -128,6 +128,11 @@ Improvements to clang-tidy
   as a value for `-export-fixes` to export individual yaml files for each
   compilation unit.
 
+- A new option `readability-identifier-naming.CheckAnonFieldInParent` is added. When set
```

Comment:
Please keep alphabetical order (by check name) in this section. If entry for `readability-identifier-naming` exists, append statement to it.